packages/frontend/src/components/common/EditableHeader.tsx

```diff
@@ -60,6 +60,7 @@ const EditableHeader: FC<Props> = ({
                     onChange={setTitle}
                     onCancel={onCancel}
                     onEdit={() => setIsEditable(true)}
+                    confirmOnEnterKey={false}
```

Comment:
This component is used in the explore to edit saved chart name. Do we want to apply this behaviour to there as well ?

Reply:
Beside this question the PR looks good 👍

Reply:
Actually this might not be necessary anymore, Good catch! 👀